packages/mobile/src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -451,11 +452,6 @@ const styles = StyleSheet.create({
     ...fontStyles.h1,
     flex: 1,
   },
-  linkArrow: {
-    height: 20,
-    width: 20,
-    marginLeft: Spacing.Regular16,
```

Comment:
Just wondering if we should leave this margin, so the description text isn't too close to the link arrow.

Reply:
I added a right margin on the text to keep the same distance :)